### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by takahiro-itou**
Hi! I made Japanese translation!
However, I translated only the my frequently using parts.
Specifically, only the menu of the main window (SandMan), context (right-click) menu, and the "File Recovery" dialog. Sorry, Other text is remain English.
Can I submit P.R. for this (incompleted) translation?

P.S.
I would like to continue translation later days.

I read the "Localization notes and tips" carefully. But if some is wrong, please let me know.

Sincerely yours,

----
Thank you for your contribution to the Sandboxie repository.

In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request" option. 

In addition, you can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".

All new translators are encouraged to look at the "Localization notes and tips" before creating a pull request: https://git.io/J9G19

### CODE REVIEW

**Comment by isaak654** [comment]
In my opinion, there are still a few things to add in the translation:

1. [Here](https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Languages.iss) you can find the location of the Sandboxie Plus installer translation file (recommended if you have already translated Plus interface)

2. [Here](https://github.com/sandboxie-plus/Sandboxie/blob/6e830b9c0a00588f72120d9db2ef559bc76e210b/Installer/Sandboxie-Plus.iss#L222-L242) you can find the location of the Sandboxie Plus installer script with the language value to add (recommended if you have already translated Plus interface)

3. [Here](https://github.com/sandboxie-plus/Sandboxie/tree/master/SandboxiePlus/SandMan/Troubleshooting) you can find the location of the Sandboxie Plus translation files for the troubleshooting wizard (lang_*.json)

4. Sandboxie Classic translation files are also used in specific Plus options, so it's advisable to look at the reports [here](https://github.com/sandboxie-plus/Sandboxie/tree/master/Sandboxie/msgs/report) to track the new entries.
   - According to [Report-Japanese.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/report/Report-Japanese.txt), there is a list of entries that should be added or commented out in [Text-Japanese-1041.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Text-Japanese-1041.txt)
   
I would say the first two are most important, while you can make the other changes in a second pull request.

**Comment by bastik-1001** [comment]
There are strings in the translation values of sandman_ja.ts that are empty and not marked as unfinished, like line 704, 709, 719, 724, 729, 755, 816, 824, 834, 839, 849, 854, 942, 947, 1032, 1037, 1042, 1047, 1052, 1057, 1083, 1088, 1093, 1098, 1106, 1111, 1145, 1163, 1168, 1174, 1272, 1277, 1282, 1296, 1306, 1311, and so on.

**Comment by takahiro-itou** [comment]
Thanks comments!
I fixed according to above two comments.

First, I re-ran lupdate and refreshed sandman_ja.ts file. This fixed the errors pointed out by @bastik-1001.
Next, I updated Installer scripts. It fixed 1. and 2. of @isaak654 comments.

**Comment by takahiro-itou** [comment]
Inno Setup has already offical Japanese translation (6.1.0+).
https://raw.githubusercontent.com/jrsoftware/issrc/main/Files/Languages/Japanese.isl

So, I use it, instead of I write translation myself.

In advance, I tested with small code using Inno Setup 6.2.2:
https://github.com/takahiro-itou/InnoSetupSample/blob/master/Sample1.iss

and I confirmed that the following line work fine!
Name: "japanese"; MessagesFile: "compiler:Languages\Japanese.isl"

**Review by isaak654 — CHANGES_REQUESTED** [review]
The following lines need to be translated exactly as done below for the other languages:
https://github.com/sandboxie-plus/Sandboxie/blob/122ae1ee87c22d4435a2bc210b9788eaf786e87c/Installer/Languages.iss#L46-L67

File: Installer/Languages.iss [hunk 1]
```diff
@@ -34,7 +34,7 @@ Name: "vietnamese"; MessagesFile: "isl\Vietnamese.isl"
 ;Name: "finnish"; MessagesFile: "compiler:Languages\Finnish.isl"
 ;Name: "hebrew"; MessagesFile: "compiler:Languages\Hebrew.isl"
 ;Name: "icelandic"; MessagesFile: "compiler:Languages\Icelandic.isl"
-;Name: "japanese"; MessagesFile: "compiler:Languages\Japanese.isl"
+Name: "japanese"; MessagesFile: "compiler:Languages\Japanese.isl"
```
Comment (isaak654) [inline Installer/Languages.iss:37]: This should be moved before the Korean line.

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by takahiro-itou** [comment]
Thanks your advices.
I fixed the two things you pointed-out above.

**Review by isaak654 — APPROVED** [review]

**Pull request merged**